CMakeLists.txt

```diff
@@ -838,4 +840,10 @@ if (UNIX AND NOT APPLE)
         find_package(OpenSSL REQUIRED)
         target_link_libraries(shadps4 PRIVATE ${OPENSSL_LIBRARIES})
     endif()
+endif()
+
+if(ENABLE_QT_GUI)
+    add_subdirectory(externals/discord-rpc)
+    target_link_libraries(shadps4 PRIVATE discord-rpc)
+    include_directories(externals/discord-rpc/include)
```

Comment:
You should use `target_include_directories` and set `BUILD_EXAMPLES` to `OFF`. And also probably put all this, except `target_link_libraries`, in `externals/CMakeLists.txt`.

Reply:
Sorry it's my first time with Cmake. Does ENABLE_QT_GUI check works inside externals/CmakeLists.txt ?

Reply:
Yes, variables are inherited in subdirectories by default.